src/NuGetGallery.Core/GitHub/RepositoryInformation.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+
+namespace NuGetGallery.GitHub
+{
+    public class RepositoryInformation : IEquatable<RepositoryInformation>, IComparable<RepositoryInformation>
+    {
+        [JsonIgnore]
+        public string Name { get; set; }
+        [JsonIgnore]
+        public string Owner { get; set; }
+        public string Url { get; set; }
+        public int Stars { get; set; }
+        public string Id
+        {
+            get => Owner + "/" + Name; set
+            {
+                var split = value.Split('/');
+                if (split.Length == 2)
+                {
+                    Owner = split[0];
+                    Name = split[1];
+                }
+            }
+        }
+
+        public List<string> Dependencies { get; set; } = null;
+
+        public RepositoryInformation()
+        { }
+
+        public RepositoryInformation(string owner, string repoName, string cloneUrl, int starCount, List<string> dependencies)
+        {
+            Owner = owner;
+            Name = repoName;
+            Url = cloneUrl;
+            Stars = starCount;
+            Dependencies = dependencies;
+        }
+
+        public override bool Equals(object obj)
```

Comment:
Don't forget to remove `Equals`, `GetHashCode`, and `CompareTo` if these are no longer needed